.travis.yml

```diff
@@ -1,6 +1,5 @@
+sudo: false
 language: node_js
 node_js:
   - "4.1"
-before_script:
-  - npm install -g pawelgalazka/runjs
-script: run test
\ No newline at end of file
+script: "npm run build"
```

Comment:
This will only build the library but not test it. Travis will try to run `npm test` by default which is not present.